android/app/src/main/res/layout/activity_speakers.xml

```diff
@@ -30,13 +30,13 @@
 
             <FrameLayout
                 android:layout_width="match_parent"
-                android:layout_height="match_parent"
+                android:layout_height="@dimen/item_image_view_extra_large"
```

Comment:
Using static height for CollapsingToolbar is alright, but doing so everywhere is not. Match Parent should be sufficient for the behaviour required here